README.md

```diff
@@ -114,8 +116,17 @@ To run the testing script, which shouts if we fell out of conformance with Micro
 
 ## 🔭 Future Features<a name="future-features"></a>
 
+- [x] Project 1: Setup and mobile version skeleton
+- [x] Project 2: Mobile version
+- [x] Project 3: Contact form (pair programming)
+- [ ] Project 4: Desktop version
+- [ ] Project 5: Application deploy
+- [ ] Project 6: Evaluate accessibility (pair programming)
+- [ ] Project 7: Mobile menu (pair programming)
+- [ ] Project 8: Details modal (pair programming)
+- [ ] Project 9: Validate contact form (pair programming)
+- [ ] Project 10: Preserve data in the browser (pair programming)
```

Comment:
Great job including the "Future Features" section in the README file! This section is important as it gives readers insight into the project's potential growth and development. However, I've noticed that the elements listed in this section are either already implemented or currently being worked on. The purpose of the "Future Features" section is to highlight upcoming functionalities that will be added in the future. To improve the section, please consider updating it to include only one to three features that are planned for implementation in the future. This way, readers can easily identify the areas where the project is expected to expand and evolve.

Maintaining a clear distinction between existing features and planned future enhancements will enhance the readability and overall usefulness of the "Future Features" section. Keep up the good work, and I'm looking forward to seeing how the project progresses! 🚀📈